examples/arrays/arrays.bal

```diff
@@ -6,10 +6,25 @@ public function main() {
     io:println(a.length());
 
     // Creates an array specifying an array literal.
-    int[] b = [1, 2, 3, 4, 5, 6, 7, 8];
+    int[] b = [1, 3, 2, 4, 7, 6, 5, 8];
```

Comment:
Shall we leave the elements as they are? Any particular reason to change?

Reply:
I initially used .sort() function as an example and then later decided not to. I will revert it back.